app/components/ui/forms.tsx

```diff
@@ -98,11 +98,26 @@ export function Button({ children, ...props }: ButtonProps) {
 type LinkButtonProps = {
   children: ReactNode;
   to: string;
+  preventScrollReset?: boolean;
+  prefetch?: LinkProps['prefetch'];
 };
 
-export function LinkButton({ to, children, ...props }: LinkButtonProps) {
+export function LinkButton({
```

Comment:
NIT: Not sure if the `forms.tsx` file is the right place for a reusable anchor tag component that looks like a button... Maybe `links.tsx`?